CardanoSharp.Wallet/Models/Addresses/Address.cs

```diff
@@ -21,6 +23,8 @@ public Address(string prefix, byte[] address)
         {
             _bytes = address;
             _address = Bech32.Encode(address, prefix);
+
+            IsValid = Bech32.IsValid(_address);
```

Comment:
just curious, should we rather throw an exception here or is having and populating the isvalid field more inline with the general style or is it part of a cddl or something?